superset/db_engine_specs/base.py

```diff
@@ -686,13 +685,14 @@ def df_to_sql(
 
     @classmethod
     def convert_dttm(  # pylint: disable=unused-argument
-        cls, target_type: str, dttm: datetime,
+        cls, target_type: str, dttm: datetime, **kwargs: Any,
```

Comment:
Is there a reason this is `**kwargs: Any` and not `db_extra: Optional[Dict[str, Any]]` like has been done in `get_column_spec`? Unpacking into kwargs will make more difficult to add new parameters to this method going forward

Reply:
Indeed, the writing here is not strict enough, for `db_extra`, it is better to show the declaration type, I think, the method signature can be like this, `def convert_dttm(cls, target_type: str, dttm: datetime, db_extra: Optional[Dict[str, Any]], **kwargs: Any)` also continue to retain `kwargs`, so that later if a data source needs non-db_extra information, this way is also compatible, what do you think

Reply:
@aniaan I'd prefer not to add `**kwargs` unless it's currently needed (it's easy enough to add later when the need comes up). And thinking more closely, I'd personally prefer to keep all arguments named so that all parameters in the method are explicit.

Reply:
Yes, we can, then we will not add it for now, and we will consider it later if we have this situation.
I'll correct the PR later

Reply:
@villebro I have updated it, you can review it when you have time